bin/cli.js

```diff
@@ -15,7 +15,7 @@ const pkgUp = require('pkg-up');
 function insideProject() {
   let nearestPackagePath = pkgUp.sync();
 
-  if (nearestPackagePath === null) {
+  if (!nearestPackagePath) {
```

Comment:
This was necessary, as it returned `undefined` for me (instead of `null`) *locally only*, so all tests were broken, even on master. Idk why? 🤷‍♂️